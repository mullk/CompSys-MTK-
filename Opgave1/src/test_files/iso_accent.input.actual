test_files/iso_accent.input: ISO-8859 text
