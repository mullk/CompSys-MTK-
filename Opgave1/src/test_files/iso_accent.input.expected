test_files/iso_accent.input: Unicode text, UTF-8 text
